homeassistant/components/waqi/sensor.py

```diff
@@ -142,7 +142,9 @@ def name(self):
     def native_value(self):
         """Return the state of the device."""
         if self._data is not None:
```

Comment:
This check can be removed, right? The `available()` property already checks for this.